webview-ui/src/components/chat/chat-view/components/layout/WelcomeSection.tsx

```diff
@@ -26,8 +27,9 @@ export const WelcomeSection: React.FC<WelcomeSectionProps> = ({
 	const shouldShowInfoBanner = lastDismissedInfoBannerVersion < CURRENT_INFO_BANNER_VERSION
 	// const shouldShowNewModelBanner = lastDismissedModelBannerVersion < CURRENT_MODEL_BANNER_VERSION
 
-	// Show CLI banner if not dismissed
-	const shouldShowCliBanner = lastDismissedCliBannerVersion < CURRENT_CLI_BANNER_VERSION
+	// Show CLI banner if not dismissed and platform is VSCode (not JetBrains/standalone)
+	const shouldShowCliBanner =
```

Comment:
If we are only showing the setting to MacOS users only, don't we need to check that here as well?

Reply:
WE should for sure, good catch.